src/App.ts

```diff
@@ -12,7 +12,6 @@ import {
   matchConstraints,
   onlyCommands,
   matchCommandName,
-  matchFunctionCallback,
```

Comment:
Removed due to reverted commit included in history after bad rebase. The changed version should match `main`. To verify: [L15](https://github.com/slackapi/bolt-js/blob/237a503d1e19a8e40ca9ed130dacab1be9752d53/src/App.ts#L15)